packages/@coorpacademy-components/src/organism/mooc/image-slider/index.js

```diff
@@ -4,10 +4,9 @@ import map from 'lodash/fp/map';
 import Slide from '../../../atom/slide';
 import Slider from '../../slider';
 
-const ImageSlider = ({slides}) => {
-  const myslides = map(slide => <Slide key={slide.title} {...slide} />, slides);
-
-  return <Slider>{myslides}</Slider>;
+const ImageSlider = ({slides = []}) => {
```

Comment:
Is it still used ?

Reply:
Dunno, but in `components ` we never delete unused components